### DISCUSSION TITLE
What is the prefix and post fix notation of (a + b) * (c + d) ?

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#34] [Q&A] [answered]
Solve for both prefix and postfix

**Comment by fcasamento** [comment, marked as answer]
Hi, @ankit-akash. I found this documentation very well explained, see if it meets the resolution you need.

https://panda.ime.usp.br/panda/static/pythonds_pt/02-EDBasicos/InfixPrefixandPostfixExpressions.html#:~:text=Prefix%20expression%20notation%20requires%20that,%2B%20A%20*%20B%20C%20in%20prefix.

**Reply by fcasamento** [reply]
@ankit-akash Mark this as an **answer** as well, we can help each other. 🙏🏻 Pls
